.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 before_install:
   - npm -g install npm@6
 node_js:
-  - "12"
```

Comment:
why is node 12 removed?

Reply:
@curcuz  B/c gulp coffeescript ended up being incompatible with it. We should completely drop it in favor on just TS and npm scripts.
See: https://travis-ci.org/balena-io-modules/balena-device-status/jobs/602757865#L309

Also had to do this in another package recently.
See: https://github.com/balena-io-modules/balena-image-manager/commit/c2618f4e55eacc79e1ed9883d1560c0d5009e905